test/unit-test/test-Model.js

```diff
@@ -4491,6 +4491,340 @@ describe('Unit Test/Model Test', function() {
       });
     });
 
+    it('"input0 as a tensor (rank = 4) of TENSOR_FLOAT32 type, input1 and input2 as INT32 type"  new_height and new_width greater than original data are ok for "RESIZE_BILINEAR" operation', () => {
+      return nn.createModel(options).then((model) => {
+        let batches = 2;
+        let height = 3;
+        let width = 4;
+        let depth = 5;
+        let new_height = 6;
+        let new_width = 6;
+        model.addOperand({type: nn.TENSOR_FLOAT32, dimensions: [batches, height, width, depth]});
+        model.addOperand({type: nn.INT32});
+        model.setOperandValue(1, new Int32Array([new_height]));
+        model.addOperand({type: nn.INT32});
+        model.setOperandValue(2, new Int32Array([new_width]));
+        model.addOperand({type: nn.TENSOR_FLOAT32, dimensions: [batches, new_height, new_width, depth]});
+        assert.doesNotThrow(() => {
+          model.addOperation(nn.RESIZE_BILINEAR, [0, 1, 2], [3]);
+        });
+      });
+    });
+
+    it('"input0 as a tensor (rank = 4) of TENSOR_FLOAT32 type, input1 and input2 as INT32 type"  new_height and new_width less than original data are ok for "RESIZE_BILINEAR" operation', () => {
+      return nn.createModel(options).then((model) => {
+        let batches = 4;
+        let height = 4;
+        let width = 4;
+        let depth = 4;
+        let new_height = 2;
+        let new_width = 2;
+        model.addOperand({type: nn.TENSOR_FLOAT32, dimensions: [batches, height, width, depth]});
+        model.addOperand({type: nn.INT32});
+        model.setOperandValue(1, new Int32Array([new_height]));
+        model.addOperand({type: nn.INT32});
+        model.setOperandValue(2, new Int32Array([new_width]));
+        model.addOperand({type: nn.TENSOR_FLOAT32, dimensions: [batches, new_height, new_width, depth]});
+        assert.doesNotThrow(() => {
+          model.addOperation(nn.RESIZE_BILINEAR, [0, 1, 2], [3]);
+        });
+      });
+    });
+
```

Comment:
Pls add Cuiyan's ['output aspect ratio is not the same as input aspect ratio'](https://github.com/cuiyanx/webml-polyfill-1/blob/add-test-cases-for-resize-bilinear-op/test/unit-test/test-Model.js#L4506) test case